src/utils/navigation.ts

```diff
@@ -132,7 +132,7 @@ export const NAVIGATION: Navigation = {
   },
   [config.OVERSEAS_NAME_WITH_PARAMS_URL]: {
     currentPage: config.OVERSEAS_NAME_PAGE,
-    previousPage: () => config.INTERRUPT_CARD_URL,
+    previousPage: (appData: ApplicationData, req: Request) => getUrlWithParamsToPath(config.INTERRUPT_CARD_WITH_PARAMS_URL, req),
```

Comment:
is the appData parameter needed here? look like it isn't

Reply:
I think it's needed because it's in the method signature of the model class. If I remove it, I get a compile error on line 22 of this class.

Reply:
ah ok, leave as is then